docker-compose.yaml

```diff
@@ -130,6 +131,33 @@ services:
     entrypoint: /usr/local/bin/client --host="server-grpc" --port="8083" --implementation="connect-grpc-h2" --cert "cert/client.crt" --key "cert/client.key"
     depends_on:
       - server-grpc
+  client-connect-to-server-grpc-adapter-h1:
```

Comment:
@emcfarlane just a dumb nit, but in my #477 PR, I tried to make the names a bit more scalable and flexible once additional languages and implementations are added. I think we settled on:

`client-{client implementation / protocol}-to-server-{server implementation}-{http version}`

where client implementation / protocol would be one of:

```
grpc-go (gRPC Go client)
grpc-web (gRPC-Web client)
connect-go (Connect-Go client using Connect protocol)
connect-go-grpc (Connect-Go client using gRPC protocol)
connect-go-grpc-web (Connect-Go client using gRPC-Web protocol)
connect-web (Connect-Web client using Connect protocol)
connect-web-grpc-web (Connect-Web client using gRPC-Web protocol)
...etc
```

Can you rename these containers to be:
```
client-connect-go-to-server-grpc-adapter-h1
client-connect-go-to-server-grpc-adapter-h2
client-connect-go-grpc-web-to-server-grpc-adapter-h2:
```

That should keep things consistent once #477 lands.  Thanks!

Reply:
Yep will do!